includes/templates/admin/edit-story.php

```diff
@@ -88,6 +88,11 @@
 			'type'    => $post_type_object->name,
 		]
 	),
+	"/web-stories/v1/$stories_rest_base/{$post->ID}/lock/?" . build_query(
+		[
+			'_embed' => 'author',
+		]
+	),
```

Comment:
This was not preloaded by design. Why is it being preloaded now?

Reply:
See discussion at https://xwp.slack.com/archives/C01H2K83WD9/p1666788696201389 for an explanation.

The aim is to immediately get the current lock owner information right when opening the editor, without having to wait for the 60s interval to pass first. There's no harm in preloading the response and saving 1 HTTP request.

Note that preloading a response only stores it once. Any subsequent calls will trigger an HTTP request as expected. Just the first one is fetched from cache.